Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -555,3 +588,83 @@ void ABenchmarkGymGameModeBase::ReportAuthoritativePlayers_Implementation(const
 		}
 	}
 }
+
+void ABenchmarkGymGameModeBase::TickActorMigration(float DeltaSeconds)
+{
+	if (bHasActorMigrationCheckFailed)
+	{
+		return;
+	}
+
+	const UNFRConstants* Constants = UNFRConstants::Get(GetWorld());
+	check(Constants);
+
+	// This test respects the initial delay timer in both native and GDK
+	if (Constants->ActorMigrationCheckDelay.HasTimerGoneOff())
+	{
+		// Count how many actors hand over authority in 1 tick
+		int Delta = FMath::Abs(UXAuthActorCount - PreviousTickMigration);
+		ToBeRemovedMigrationDeltas.Enqueue(ToBeRemovedDelta(Delta, DeltaSeconds));
+		bool bChanged = false;
+		if (MigrationCountSeconds > MigrationWindowSeconds)
+		{
+			ToBeRemovedDelta RemoveValue;
+			if (ToBeRemovedMigrationDeltas.Peek(RemoveValue))
```

Comment:
Prefer use Dequeue(OldestValue) ... instead of peek then pop, also prefer the name OldestValue instead of RemoveValue, OldestValue is more clear.